.github/workflows/main.yml

```diff
@@ -58,7 +58,13 @@ jobs:
       - name: Output test failures
         # Makes it easier to find failed tests so no need to scroll through the whole log.
         if: ${{ failure() && env.TARGET_GOLANG_VERSION == matrix.go }}
-        run: cat test_results.json | jq 'select(.Action == "fail")'
+        run: |
+          cat test_results.json | jq 'select(.Action == "fail")'
```

Comment:
I don't fully understand this, what makes sure that we run `Sanitize test results` before `Output test failures`?

Reply:
@Olshansk We wanted to make sure we sanitize JSON prior to submitting it to the `guyarb/golang-test-annotations` github action. As you mention it, it probably is a good idea to also sanitize prior to exicuting `jq`'s `select()` query.

Added a change! Thanks.